pyproject.toml

```diff
@@ -1,3 +1,62 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[tool.hatch.build.targets.sdist]
+exclude = ["tests*"]
+
+[tool.hatch.version]
+path = "substra/__version__.py"
+
+[project]
+name = "substra"
+description = "Low-level Python library for interacting with a Substra network"
+dynamic = ["version"]
+requires-python = ">= 3.9"
+dependencies = [
+    "requests",
+    "urllib3<2",
```

Comment:
That's interesting 🙃 
I guess this was pinned to avoid a bug at some point, might be worth investigating if it's still necessary

Reply:
worth create a card you think ?

Reply:
yes, probably, else we'll forget about it again :)

Reply:
done